.claude/agent-memory/archgate-developer/feedback_prefer_tests_over_adr_rules.md

```
@@ -0,0 +1,18 @@
+---
+name: pick-the-right-enforcement-layer
+description: Static syntactic invariants belong in a custom oxlint rule (.archgate/lint/oxlint.ts), not in ADR .rules.ts and not in tests
+metadata:
+  type: feedback
+---
+
+Pick the enforcement layer by the nature of the invariant — don't default to ADR rules.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)